lib/traveller.rb

```diff
@@ -32,7 +32,7 @@ def initialize(input)
   end
```

Comment:
Are there any tests for this gem? If not we should probably add.

Reply:
I didn't see any. I could make a PT for that.